roles/engineblock/defaults/main.yml

```diff
@@ -51,6 +51,9 @@ engine_api_profile_password: secret
 
 engine_engine_idp_baseurlpath: "https://{{ engine_domain }}/simplesaml/"
 
+# Minimum execution time in milliseconds when response is invalid
+engine_minimum_execution_time_on_invalid_response: 5000
```

Comment:
the more I see it... `...on_invalid_received_response`? I mean it's not that EB gives an invalid response, it's that we get one that is deemed invalid...